sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallRecordingAsyncLiveTests.java

```diff
@@ -39,7 +39,7 @@ public void recordingOperations(HttpClient httpClient) {
             CallRecordingAsync callRecording = client.getCallRecordingAsync();
             RecordingStateResult recordingResponse = callRecording.startRecording(
                 new StartRecordingOptions(new ServerCallLocator(serverCallId))
-                    .setRecordingStateCallbackUri(new URI(ngrok)))
+                    .setRecordingStateCallbackUrl(ngrok))
```

Comment:
since URI is not used here, please remove unused import of URI. Otherwise the pipeline doesn't allow it to build.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=1810223&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=ba3cf43a-b2bf-50f9-f95f-93049ca44ad1